src/translator/response_builder.cpp

```diff
@@ -1,27 +1,28 @@
 #include "response_builder.h"
 #include "response_options.h"
+#include "quality_estimator.h"
 
 namespace marian {
 namespace bergamot {
 
+
 void ResponseBuilder::buildQualityScores(Histories &histories,
                                          Response &response) {
-  std::vector<Quality> qualityScores;
+  QualityEstimator model(qualityEstimator_->begin());
```

Comment:
No creating an instance every `Response`, especially if this bit becomes heavier with models like KenLM etc which seem to come up in discussions.

Reply:
done:  064cfe3